### ISSUE TITLE
Using Typescript for better type checking

### ISSUE THREAD

**Issue by qaz74107410** [thananon/twitch_tools#11]
I would suggest to refactor to Typescript instead of Javascript. No more `if(!Number(amount)){
            return false
        }`
       
Anyway, The plus is everyone could know what type of parameter can they put through the function, etc. which is really good for new contributors.

**Comment by thananon** [comment]
Yeah, good suggestion. Only thing is I dont know anything about TS. It seems people really wants to go for it. Good. Please open a PR.
